lib-rca/src/main/java/com/facebook/openwifi/librca/inputs/RCAParams.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.inputs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Define root cause analysis configuration parameters */
+public final class RCAParams {
+
+	/** Look-back window in minutes */
+	public final int detectionWindowMin;
```

Comment:
Why not use milliseconds? So that we don't need to transfer it to ms during calculation.
Just put `(6hr)` in your java doc, then it's clear enough to help developers understand how long the time window is.

Reply:
I can do ms, I'll change it